fractal_server/app/api/v1/project.py

```diff
@@ -320,6 +320,10 @@ async def apply_workflow(
         input_dataset_id=input_dataset_id,
         output_dataset_id=output_dataset_id,
         workflow_id=workflow_id,
+        workflow_dump=dict(
+            workflow.dict(),
+            task_list=[wf_task.dict() for wf_task in workflow.task_list],
```

Comment:
I think this ignores `wf_task.task`, which would not be OK.

Reply:
true, I'll add in the next commit